src/styles/partials/_typography.css

```diff
@@ -240,105 +240,6 @@ a {
 	color: var(--primary-1);
 }
 
-a:before,
-a:after {
-	position: absolute;
-	z-index: -1;
-	display: block;
-	content: '';
-}
-
-a:before {
-	top: 0;
-	left: 0;
-	width: 100%;
-	height: 100%;
-	transition: 100ms ease-out 50ms;
-	transform-origin: 0 25px;
-	background: var(--highlight-color);
-}
-
-a:after {
-	right: -4.5px;
-	bottom: 4.75px;
-	transition: 50ms ease-out;
-	transform: scaleX(1);
-	transform-origin: left center;
-	border: 7px solid transparent;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-	a:hover,
-	a:active,
-	a:focus-visible {
-		outline: none;
-		color: var(--primary-2);
-	}
-
-	a:hover:before,
-	a:active:before,
-	a:focus-visible:before {
-		transition: 100ms ease-out;
-		transform: scaleY(0.15);
-		background: var(--accent-2);
-	}
-
-	a:after {
-		right: -14px;
-		transition: transform 50ms ease-out 100ms;
-		transform: none;
-	}
-
-	a:hover:after,
-	a:active:after,
-	a:focus-visible:after {
-		border-left-color: var(--accent-2);
-	}
-}
-
-@media (max-width: 375px) {
-	a:before {
-		transform-origin: 0 17.5px;
-	}
-
-	a:after {
-		right: -10px;
-		bottom: 1px;
-		border: 5.5px solid transparent;
-	}
-}
-
-sup a {
```

Comment:
We actually need lines 311-320 so links in superscript (in effect, * references) have a different styling ;).

Reply:
ah, okay. wen't a little beyond the actual code. oopsies